substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/c/CAnnotationProcessor.java

```diff
@@ -87,13 +95,19 @@ public NativeCodeInfo process(CAnnotationProcessorCache cache) {
              * Generate C source file (the "Query") that will produce the information needed (e.g.,
              * size of struct/union and offsets to their fields, value of enum/macros etc.).
              */
-            writer = new QueryCodeWriter(tempDirectory);
+            writer = new QueryCodeWriter(queryCodeDirectory);
             Path queryFile = writer.write(codeInfo);
             if (nativeLibs.getErrors().size() > 0) {
                 return codeInfo;
             }
             assert Files.exists(queryFile);
 
+            if (CAnnotationProcessorCache.Options.DontRunCAPQueryCode.getValue()) { // Only output
```

Comment:
`mx eclipseformat` can sometimes bug up :) Put the comment above this line between /* and */ - I think that wouldn't cause it to break into 3 lines

Reply:
Should be fixed now :)